cordapp/src/main/kotlin/com.luxoft.blockchainlab.corda.hyperledger.indy/flow/CreatePairwiseFlow.kt

```diff
@@ -2,6 +2,7 @@ package com.luxoft.blockchainlab.corda.hyperledger.indy.flow
 
 import co.paralleluniverse.fibers.Suspendable
 import com.luxoft.blockchainlab.hyperledger.indy.IndyUser
+import com.luxoft.blockchainlab.hyperledger.indy.getIdentity
```

Comment:
??

Reply:
getIdentity() without arguments is now an extension function applied to the interface. This approach powers you to add a function to the interface but locks its implementation.